Cargo.toml

```diff
@@ -48,7 +48,7 @@ rustdoc-args = ["--document-private-items", "--generate-link-to-definition"]
 [dependencies]
 abscissa_core = { version = "0.8.1", default-features = false, features = ["application"] }
 rustic_backend = { version = "0.5.0", features = ["cli"] }
-rustic_core = { version = "0.6.0", features = ["cli"] }
+rustic_core = { version = "0.6.1", features = ["cli"] }
```

Comment:
can we do this in a separate PR?

Reply:
Ah, this is because of `FilePolicy`, right?

Reply:
yes